go/cmd/dolt/commands/status.go

```diff
@@ -57,14 +57,15 @@ func (cmd StatusCmd) Docs() *cli.CommandDocumentation {
 
 func (cmd StatusCmd) ArgParser() *argparser.ArgParser {
 	ap := argparser.NewArgParserWithMaxArgs(cmd.Name(), 0)
+	ap.SupportsFlag(cli.ShowIgnoredFlag, "i", "Show tables that are ignored (according to dolt_ignore)")
```

Comment:
We should stick to git's syntax wherever possible, unless there's a really compelling reason not to. For `git status`, the flag it uses to show ignored files is `--ignored`, with no shorthand `-i` allowed. (technically `--ignored` can take values as well, but they aren't required, so I'd keep it simple and not support any values yet)

Reply:
Fixed.